Cython/Compiler/ExprNodes.py

```diff
@@ -10534,7 +10534,10 @@ def type_error(self):
         self.type = PyrexTypes.error_type
 
     def analyse_cpp_operation(self, env, overload_check=True):
-        entry = env.lookup_operator(self.operator, [self.operand])
+        operand_types = [self.operand.type]
+        if not getattr(self, 'is_prefix', True):
```

Comment:
Normally when I try to sneak in `getattr` I don't succeed. Therefore I'm going to propose a slightly different approach

```suggestion
        if self.is_inc_dec_op and not self.is_prefix:
```

Reply:
just out of interest: what is the problem with getattr? Is this just a stylistic preference, or are the any problems with it?

Reply:
Mostly stylistic I think. In fairness, this version (or a similar one with is instance) you immediately know it refers to a postfix increment or decrement operator because it's all there in the if statement. While with `getattr` you have to reason about why it might not be set, and why `True` is a suitable default value. So I think it is clearer